routes/welcome.tsx

```diff
@@ -5,7 +5,7 @@ import { isGitHubSetup } from "@/utils/github.ts";
 
 function SetupInstruction() {
   return (
-    <div class="bg-green-50 dark:(bg-gray-900 border border-green-800) rounded-xl max-w-screen-sm mx-auto p-8 space-y-2">
+    <div class="bg-green-50 dark:bg-gray-900 dark:border dark:border-green-800 rounded-xl max-w-screen-sm mx-auto p-8 space-y-2">
```

Comment:
Are grouped classes no longer working?

Reply:
https://fresh.deno.dev/docs/examples/migrating-to-tailwind

> Tailwind CSS doesn’t support the grouping syntax from twind: text(lg uppercase gray-100). These need to be rewritten to their expanded values like text-lg uppercase text-gray-100.

Reply:
Oh, I wasn't aware that was a Twind thing. Cool.